app/components/Instrument.ts

```diff
@@ -71,10 +85,6 @@ export class Instrument {
         pvaddress: `${this.prefix}DAE:RUNNUMBER`,
         human_readable_name: "Run number",
       },
-      {
-        pvaddress: `${this.prefix}DAE:STARTTIME`,
-        human_readable_name: "Start number",
-      },
```

Comment:
Won't let me comment on the actual line, but `title`/`users` not showing in the table for me.

Reply:
hmm yes and i know why - the dashboard PV "steals" the update because of this: 
```
const pv =
        findPVInDashboard(currentInstrument.dashboard, updatedPVName) ||
        findPVByAddress(currentInstrument.runInfoPVs, updatedPVName) ||
        findPVInGroups(
          currentInstrument.groups,
          currentInstrument.prefix,
          updatedPVName,
        );
```

as the dashboard is before the runinfo. 

will have a think. maybe it's not useful having it twice anyway?

Reply:
I'm fine with deleting it here if it's always part of the top panel anyway.